modules/Sources/Dependencies/UserNotificationCenter/Models/NotificationSettings.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  File.swift
```

Comment:
```suggestion
//  NotificationSettings.swift
```
I can't believe it's 2023 and Xcode has this bug still